raiden/tests/unit/test_messages.py

```diff
@@ -270,3 +288,100 @@ def test_refund_transfer_invalid_values(invalid_values):
     for invalid_value in invalid_values:
         with pytest.raises(ValueError):
             factories.create(factories.RefundTransferProperties(**invalid_value))
+
+
+def assert_method_call(mock, method, *args, **kwargs):
+    child_mock = getattr(mock, method)
+    child_mock.assert_called_once_with(*args, **kwargs)
+    child_mock.reset_mock()
+
+
+def test_message_handler():
```

Comment:
The test has more than 80 lines, so a docstring would help a lot.

Reply:
I added one, though the test is quite straightforward and there wasn't much to say. An alternative would be to move the setup into fixtures and split the test into several small tests.

Reply:
I thought it was a long scenario of some sort.  So the docstring removed my misconception.